x/ibc-transfer/types/trace.go

```diff
@@ -20,62 +21,49 @@ import (
 //
 // Examples:
 //
-// 	- "portidone/channelidone/uatom" => DenomTrace{Trace: "portidone/channelidone", BaseDenom: "uatom"}
-// 	- "uatom" => DenomTrace{Trace: "", BaseDenom: "uatom"}
+// 	- "portidone/channelidone/uatom" => DenomTrace{Path: "portidone/channelidone", BaseDenom: "uatom"}
+// 	- "uatom" => DenomTrace{Path: "", BaseDenom: "uatom"}
 func ParseDenomTrace(rawDenom string) DenomTrace {
 	denomSplit := strings.Split(rawDenom, "/")
 
 	if denomSplit[0] == rawDenom {
 		return DenomTrace{
-			Trace:     "",
+			Path:      "",
 			BaseDenom: rawDenom,
 		}
 	}
 
 	return DenomTrace{
-		Trace:     strings.Join(denomSplit[:len(denomSplit)-1], "/"),
+		Path:      strings.Join(denomSplit[:len(denomSplit)-1], "/"),
 		BaseDenom: denomSplit[len(denomSplit)-1],
 	}
 }
 
 // Hash returns the hex bytes of the SHA256 hash of the DenomTrace fields using the following formula:
 //
-// hash = sha256(trace + "/" + baseDenom)
+// hash = sha256(tracePath + "/" + baseDenom)
 func (dt DenomTrace) Hash() tmbytes.HexBytes {
-	return tmhash.Sum([]byte(dt.GetPrefix() + dt.BaseDenom))
+	return tmhash.Sum([]byte(dt.GetFullDenomPath()))
 }
 
 // GetPrefix returns the receiving denomination prefix composed by the trace info and a separator.
 func (dt DenomTrace) GetPrefix() string {
-	return dt.Trace + "/"
+	return dt.Path + "/"
 }
 
-// IBCDenom a coin denomination for an ICS20 fungible token in the format 'ibc/{hash(trace +
+// IBCDenom a coin denomination for an ICS20 fungible token in the format 'ibc/{hash(tracePath +
 // baseDenom)}'. If the trace is empty, it will return the base denomination.
```

Comment:
nit suggestion
```suggestion
// IBCDenom a coin denomination for an ICS20 fungible token in the format
// 'ibc/{hash(tracePath + baseDenom)}'. If the trace is empty, it will return the base denomination.
```